packages/components/src/styles/common-styles.ts

```diff
@@ -125,9 +157,9 @@ export const getBackfaceVisibilityJssStyle = (): JssStyle => ({
  */
 export const getFrostedGlassBackgroundJssStyles = (
   isVisible: boolean,
-  duration: string,
+  duration: MotionDurationKeyFinal,
   theme: Theme,
-  timingFn = 'cubic-bezier(.16,1,.3,1)'
+  timingFn: MotionEasingKeyFinal = 'base'
```

Comment:
Is this parameter ever assigned or this its default overridden? If not, let's just remove it.